lib/must/util.go

```diff
@@ -0,0 +1,11 @@
+package must
```

Comment:
What's the purpose of the `must` package? That it panics? Why not just use `url.Parse` as-is and handle any errors?